launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/run_out.param.yaml

```diff
@@ -30,9 +30,12 @@
         enable: false
         margin: 0.0           # [m] distance on how close ego approaches the obstacle
         limit_vel_kmph: 3.0   # [km/h] limit velocity for approaching after stopping
-        stop_thresh: 0.01     # [m/s] threshold to decide if ego is stopping
-        stop_time_thresh: 3.0 # [sec] threshold for stopping time to transit to approaching state
-        dist_thresh: 0.5      # [m] end the approaching state if distance to the obstacle is longer than stop_margin + dist_thresh
+
+      # parameters for the change of state. used only when approaching is enabled
+      state:
+        stop_thresh: 0.01          # [m/s] threshold to decide if ego is stopping
+        stop_time_thresh: 3.0      # [sec] threshold for stopping time to transit to approaching state
+        disable_approach_dist: 4.0 # [m] end the approaching state if distance to the obstacle is longer than this value
```

Comment:
Is it ok that `disable_approach_dist` has a different value than before?
Before this PR, stop_margin + dist_thresh = 2.5 + 0.5 = 3.0

Reply:
Thank you, it was a mistake.
fixed https://github.com/autowarefoundation/autoware.universe/pull/1884/commits/659e4d6f6c4e6d28c1cbc6c5b062b8d2c43becd5